index.html

```diff
@@ -112,9 +112,33 @@ <h2>A project for first-time contributions</h2>
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ================================================ -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
-        <!-- Keep one line of space above and below your card -->
-        <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
```

Comment:
You have removed these 2 lines of code, which you were not supposed to do so.
So please add those two line and then commit and push your changes.
We will merge it after reviewing it. If you have any doubt or need any help regarding this feel free to ask.
Thank you

Reply:
> You have removed these 2 lines of code, which you were not supposed to do so. So please add those two line and then commit and push your changes. We will merge it after reviewing it. If you have any doubt or need any help regarding this feel free to ask. Thank you

Done changes